browser/components/SimpleContributions/steps/Agreement.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import ExtensionPoint from '../../../util/ExtensionPoint';
+import SurveyStep from '../SurveyStep';
+
+interface State {
+  ready: boolean;
+}
+
+export default class Agreement extends SurveyStep<State> {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      ready: false,
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ready: true});
+    }, 1 * 1000);
+  }
+
+  render() {
+    return (
+      <div>
+        <ExtensionPoint ext="simple-contrib-landing-dialog">
```

Comment:
Honestly, I'd leave this whole section blank or with a "To be filled in by administrator" thing.

Reply:
I wanted to add hooks for easy usage but also have a default dialog that made sense.